### DISCUSSION TITLE
v2 EOL. v3 Anouncement

### DISCUSSION THREAD

**Discussion by clsource** [jasonelle-archive/jasonelle-v2#120] [General]
Hello I think is time to anounce that my plans for Jasonette v2 
is archiving it at the end of this year (December 2022). 
After four happy years of maintaining Jasonette after Ethan's left.

- There are lot of unkept dependencias both in Android and iOS
- I don´t have enough bandwidth to support Android
- I am making a v3 from scratch (clean room).

Please if you wish to take the lead and be maintainer for Jasonette v2,  contact me at the jasonelle telegram

https://t.me/jasonelle

## What will be new for v3?

- Will focus only on iOS (Android not planned, maybe in the future but there will be no promises)
- Will have a modular rendering engine (that way it would be easier to add new components)
- Will use Swift (when it makes sence to do so, still will be using Objecive-C in some parts)
- [BREAKING CHANGE] Not backwards compatible with Jasonette v2  (but a rendering engine can be created for this in the future)
- [BREAKING CHANGE] Minimalist and modular. Just the bare minimum to start something. Extensions can be created to bring more power.
- [BREAKING CHANGE] Javascript based, instead of Json based. (Still will be familiar to Jasonette users).
- No Cocoapods.
- More goodies
- Estimate time: Not clear. But there is already something in the works.

So, it basically will be a totally different product that is inspired by Jasonette ideas and functions.
It will still target people that loves low code environments.

My vision is a cozy framework to do experiments in iOS platform.
Maybe someone can use these ideas and make something for Android too :)

For people that would like to test some alternatives in the meantime.

- https://jsonnet.org/ (Similar to the selecttransform engine)
- https://adaptivecards.io/
- https://nexusdev.tools/
- https://hyperview.org/
- https://www.componentbox.io/

**Comment by dumblob** [comment]
@clsource I think those are wise decisions. Do you have any repo with "v3" I could follow and occassionally comment and/or ask on/about things?

**Reply by clsource** [reply]
For now it's private until an open beta 👍

**Comment by Jleagle** [comment]


**Reply by clsource** [reply]
yes would be a new repo. The old ones would be archived.
